src/main/windows/editor.js

```diff
@@ -397,6 +397,18 @@ class EditorWindow extends BaseWindow {
 
   // --- private ---------------------------------
 
+  _buildUrlString (windowId, env, userPreference) {
+    const url = this._buildUrlWithSettings(windowId, env, userPreference)
+    const {
```

Comment:
I have two question:

1. Why not put this `spellcheckerIsHunspell ` in `_buildUrlWithSettings `?
2. No need to use getAll, we can use getItem if only get one preference item.

Reply:
It's not needed for settings. The editor need to set an environment variable to force the use of Hunspell on macOS.